docs/source/getting_started/advanced.rst

```diff
@@ -116,17 +123,39 @@ learning:
 
   - Check the documentation: :py:class:`lightly.models.simsiam.SimSiam`
 
+- `Barlow Twins: Self-Supervised Learning via Redundancy Reduction, S. Deny, 2021 <https://arxiv.org/abs/2103.03230v1>`_
+
+  - Check the documentation: :py:class:`lightly.models.barlowtwins.BarlowTwins`
+
 Do you know a model that should be on this list? Please add an issue on GitHub :)
 
 All models have a backbone component. This could be a ResNet.
 When creating a self-supervised learning model you pass it a backbone. You need
 to make sure the backbone output dimension matches the `num_ftrs` parameter 
 of the model.
 
+Lightly has a built-in generator for ResNets. However, the model architecture slightly differs from the official ResNet implementatation.
+The difference is in the first few layers. Whereas the official ResNet starts 
+with a 7x7 convolution the one from lightly has a 3x3 convolution. 
+
+* The 3x3 convolution variant is more efficient (less parameters and faster 
+  processing) and is better suited for small input images (32x32 pixels or 64x64 pixels). 
+  We recommend to use the lighlty variant for cifar10 or running the model on a microcontroller 
+  (see https://github.com/ARM-software/EndpointAI/tree/master/ProofOfConcepts/Vision/OpenMvMaskDefaults)
+* However, the 7x7 convolution variant is better suited for larger images 
+  since the number of features is smaller due to the stride and additional 
+  `MaxPool2d` layer. For benchmarking against other academic papers on 
+  datasets such as ImageNet, Pascal VOC, MOCO, etc. use the torchvision variant.
+
 .. code-block:: python
 
-        # create a ResNet backbone and remove the classification head
+        # create a lightly ResNet
         resnet = lightly.models.ResNetGenerator('resnet-18')
+
+        # alternatively create a torchvision ResNet backbone
+        resnet_torchvision = torchvision.models.resnet18()
+
+        # remove the last linear layer and add an adaptive average pooling layer
         backbone = nn.Sequential(
```

Comment:
alternatively, this can be done with
`resnet_torchvision.fc = nn.AdaptiveAvgPool2d(1)`

However, I am not sure whether we want to add it here.

Reply:
Would leave this one out to prevent confusion